invenio_communities/config.py

```diff
@@ -23,7 +23,12 @@
 
 import pkg_resources
 
-COMMUNITIES_REQUEST_EXPIRY_TIME = 7  # Default expiry time delta in days
+COMMUNITIES_REQUEST_EXPIRY_TIME = 7
```

Comment:
For consistency with other modules I would propose to use seconds to defining time.